langs/arturo/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM debian:bookworm-slim AS builder
+
+ENV PATH="/root/.nimble/bin:$PATH"
+
+RUN apt-get update && apt-get install -y curl gcc git libpcre3-dev xz-utils
+
+RUN curl https://nim-lang.org/choosenim/init.sh -sSf | sh -s -- -y && choosenim stable
+
+RUN git clone https://github.com/arturo-lang/arturo.git
```

Comment:
Rather than clone HEAD, either pass `-b xxx` or better yet skip git and just download a release tarball of the source - https://github.com/arturo-lang/arturo/archive/refs/tags/v0.9.83.tar.gz

This should fix your version number issue by pinning the source to a known version.

Reply:
Yeah, so the tarball, or at least what's in it, is not behaving very nice. I will take care of that later today, guaranteed, because I have to be here and go there and I'm about to go nuts if I take care of it at this particular moment. I made sure it gets masked in the build process, but it'll be taken care of. Yet another good exercise, battling tarballs. I will ship the final commit momentarily.

Reply:
But, I went Alpine already! 😃

Reply:
No matter what, the build process of 0.9.83 abruptly terminates shortly after it's initiated.

![image](https://github.com/user-attachments/assets/17182235-33cd-44f3-a7ba-1781d424591a)

I've had a few of those, which I was able to resolve. This one, I don't think so, and I created a post on the project's Discord for additional information.

Reply:
Eh, the `nim` command has a million command-line options. Time to play!

Reply:
```bash
oh@Yewzir:~/code-golf$ docker build -t codegolf/lang-arturo langs/arturo
[+] Building 73.5s (21/21) FINISHED
.
.
.
oh@Yewzir:~/code-golf$ docker run -it --rm codegolf/lang-arturo
arturo v/0.9.83 (amd64/linux)
```

Incoming.